docs/staging/freezes/freeze-exceptions.md

```diff
@@ -0,0 +1,101 @@
+(freeze-exceptions)=
+# Freeze exceptions
```

Comment:
There are many freeze types and that makes this ambiguous
Please call them clearly feature freeze exceptions